src/features/commands.ts

```diff
@@ -638,47 +638,119 @@ Instead:
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+
+      } else {
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: false,
+              CREATE_PUBLIC_THREADS: false,
+              CREATE_PRIVATE_THREADS: false,
+              SEND_MESSAGES_IN_THREADS: false,
+            },
+          );
+          guildTextChannel.send("This channel has been locked by a moderator");
+        }
       }
     },
   },
   {
     words: ["!unlock"],
-    help: "",
-    category: "Communication",
+    help: "used for unlocking channel",
+    category: "Moderation",
     handleMessage: async (msg) => {
       if (!msg.guild || !isStaff(msg.member)) {
-        return;
+        await msg.react("💩")
+        await msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+      } else {
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: null, // null will inherit the permission from the parent channel category
+            },
+          );
+        }
       }
-
-      // permission overwrites can only be applied on Guild Text Channels
-      if (msg.channel.type === "GUILD_TEXT") {
-        const { channel: guildTextChannel } = msg;
-        guildTextChannel.permissionOverwrites.create(
-          guildTextChannel.guild.roles.everyone,
-          {
-            SEND_MESSAGES: null, // null will inherit the permission from the parent channel category
-          },
-        );
+    },
+  },
+  {
+    words: ["!ban"],
+    help: "used for banning users \n\t(usage: `!ban reason user`)",
+    category: "Moderation",
+    handleMessage: (msg) => {
+      if (!msg.guild || !isStaff(msg.member)) {
+        msg.react("💩")
+        msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
```

Comment:
Same reasoning as above, silently failing for users who can't use it is fine. We also definitely do not want a message this rude -- "don't try to do this this kind of shit" is a pretty hostile way to say this
```suggestion
        return;
```

Reply:
After reading this message no one will dare to run it again in a funny manner because its posted by bot

Reply:
@nickmccurdy whats your opinion ???

Reply:
It's even better if we don't allow users who aren't authorized to know this command exists.  It's information leakage

Reply:
It won't affect how people will react to it, and if i will show to it publicly, people will get that if they left for some time the trick won't work

Reply:
And how you would hide opensource code ???

Reply:
The idea is that:

a) it just shouldn't react at all when a non-staff triggers the command
b) it's not polite/professional to use "this kind of shit" as a message

Reply:
It doesn't matter if it's posted by a bot or not. It's unprofessional and only funny if you are 14 years old.

Reply:
I am 11 (i think your refering my age)

Reply:
Or let it be, any suggestions for saying same thing in polite manner ??

Reply:
I'm referring to this being immature in general. 

IMO the command shouldn't output anything to non-mod users. Additionally, I would assume that mods would use it in a non-public channel anyway to avoid spam.

Reply:
Ok, i will remove it